encrypted-media-respec.html

```diff
@@ -1240,7 +1259,31 @@
                         </dd>
                         <dt><a def-id="persistent-usage-record-session"></a></dt>
                         <dd>
-                          <p>Let <var>requested license type</var> be a non-persistable license that will persist a <var>record of key usage</var>.</p>
+                          <ol>
+                            <li>
+                              <p>Initialize this object's <var>record of key usage</var> as follows.</p>
+                              <ul>
+                                <li>
+                                  <p>
+                                    Set the list of <a def-id="key-id">key IDs</a> <a href="#known-key">known</a> to the session to the empty list.
```

Comment:
Well, certainly mathematically speaking there is only one empty list just as there is only one empty set, only one number zero etc. I'll admit that in Javascript one can have many distinct objects where the *value* of each one is the empty list. The "list of key ids known to the session" is an abstract (unspecified) kind of list, rather than a concrete type such as a Javascript Array object, so I think what is written would be ok, but - since we are in abstract type land - "an" would be ok too and may be potentially less confusing.